rest_framework/pagination.py

```diff
@@ -201,7 +201,34 @@ def to_html(self):  # pragma: no cover
         raise NotImplementedError('to_html() must be implemented to display page controls.')
 
 
-class PageNumberPagination(BasePagination):
+class PageSizePaginationMixin(object):
```

Comment:
Side point: I would avoid mixins wherever possible, even if that means duplicating code.

We only ever use mixins in REST framework in a *very* restricted way.